dream_demo/bin/main.eml.ml

```diff
@@ -15,90 +12,10 @@ let generate_uuid () = Uuidm.(v `V4 |> to_string)
 
 let add_dog name breed =
   let id = generate_uuid () in
-  let dog = { id; name; breed } in
+  let dog = { Dog.id; name; breed } in
```

Comment:
Need to specify module here as well.

Reply:
Why is that needed on the `id` field, but not on `name` or `breed`?